### ISSUE TITLE
Master Face Recognition (Deep Learning)

### ISSUE THREAD

**Issue by plusminuschirag** [omi10859/jarvis#41]
**Using Facial Recognition to be applied as a security system in Groot.**
- Setting up facial lock on Groot.
- Using Facial Recognition to show data to Master User Only.

**Comment by plusminuschirag** [comment]
I will take up this issue. Please assign it to me.

**Comment by omi10859** [comment]
``Setting up facial lock on Groot.``
But why?

**Comment by plusminuschirag** [comment]
To fully use cache mechanism and avoid logging in multiple times in same session.

**Comment by plusminuschirag** [comment]
Any Update @omi10859

**Comment by omi10859** [comment]
Normal Groot will be used in the system and everyone uses screen locks and anyway Groot doesn't have any sensitive data for now.

**Comment by lalitmee** [comment]
Yeah, I think locking Groot doesn't make sense since it doesn't have any information to hide. @iamchiragsharma, I think if you can tell your main idea behind the feature that would be a great help in understanding what you are trying to say?

**Comment by plusminuschirag** [comment]
@lalitmee @omi10859 
We are using credentials for google drive in the note making feature.
All of the apis which are going to make Groot easy for the user will be holding credentials. 
There are two problems right now:
1. Every time we create note we create new instance of authorization. We can leverage authorization as one time using Face Detection. So for a single line note you have to login again and again.
2. If we extend this to use various apis. This feature is going to act as single log-in for all the accounts.
